sycamore/llms/llms.py

```diff
@@ -10,7 +10,7 @@
 
 class OpenAIModels(Enum):
     TEXT_DAVINCI = "text-davinci-003"
-    GPT_3_5_TURBO = "gpt-3.5-turbo"
+    GPT_3_5_TURBO = "gpt-3.5-turbo-1106"
```

Comment:
Thinking about how we want to handle model changes like this. Looks like OpenAI will eventually point gpt-3.5-turbo to gpt-3.5-turbo-1106, so I guess it's fine to upgrade like this, though maybe we should expose the different variants to try to limit surprising behavior changes? I don't think it's a huge deal for this PR, but as the number of models increases we will need to come up with a standard approach here.

Reply:
This will auto-upgrade in eight days, so I think it may make sense to test the performance on this branch and then force the change. But also happy to reverse this patch and deal with this on a separate branch.